### PR TITLE
Using ParameterExclusionList instead of ParameterFilterBlackList

### PR DESCRIPTION
**Pull request by paganotoni**
cc: @markbates

### CODE REVIEW

File: middleware/param_logger.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type parameterLogger struct {
 // ParameterLogger logs form and parameter values to the loggers
 func ParameterLogger(next buffalo.Handler) buffalo.Handler {
 	pl := parameterLogger{
-		blacklist: ParameterFilterBlackList,
+		blacklist: ParameterExclusionList,
```
Comment (markbates) [inline middleware/param_logger.go]: And this `blacklist`. :)

File: middleware/param_logger.go [hunk 2]
```diff
@@ -122,12 +122,12 @@ func (pl parameterLogger) addFormFieldTo(c buffalo.Context, form url.Values) err
...
 }
 
-//maskSecrets matches ParameterFilterBlackList against parameters passed in the
+//maskSecrets matches ParameterExclusionList against parameters passed in the
 //request, and returns a copy of the request parameters replacing blacklisted params
 //with [FILTERED].
 func (pl parameterLogger) maskSecrets(form url.Values) url.Values {
 	if len(pl.blacklist) == 0 {
-		pl.blacklist = ParameterFilterBlackList
+		pl.blacklist = ParameterExclusionList
```
Comment (markbates) [inline middleware/param_logger.go]: and this one too. :)

**Comment by markbates** [comment]
@paganotoni there are still a few references to `blacklist` left, can you please change them so we can merge? Thanks.

**Comment by paganotoni** [comment]
@markbates i think is covered now, please take a look and let me know if i missed something.

**Pull request merged**